cg/meta/orders/ticket_handler.py

```diff
@@ -185,27 +190,25 @@ def replace_empty_string_with_none(cls, obj: Any) -> Any:
         return obj
 
     def connect_to_ticket(
-        self, order: OrderIn, user_name: str, user_mail: str, project: str, ticket_number: str
+        self, order: OrderIn, user_name: str, project: str, ticket_number: str
     ) -> None:
         """Appends a new order message to the ticket selected by the customer"""
-        LOG.info(f"Connecting order to ticket {ticket_number}")
+        LOG.info("Connecting order to ticket %s", ticket_number)
```

Comment:
For all string building purposes we use f strings. Can you check the other instances throughout the PR and align it to that?
```suggestion
        LOG.info(f"Connecting order to ticket {ticket_number})
```